registration/views.py

```diff
@@ -26,9 +26,17 @@ class _RequestPassingFormView(FormView):
     enable finer-grained processing.
 
     """
+    def _clean_next_param(self, request):
```

Comment:
This should use Django's `is_safe_url` - see [this nice blog post from Darian Moody](https://www.djm.org.uk/djangos-little-protections-word-redirect-dangers/)